molviewspec/molviewspec/nodes.py

```diff
@@ -160,9 +162,30 @@ class DownloadParams(BaseModel):
     url: str = Field(description="URL from which to pull structure data.")
 
 
-ParseFormatT = Literal["mmcif", "bcif", "pdb"]
+ParseFormatT = Literal["mmcif", "bcif", "pdb", "map"]
+# RawVolumeSourceT = Literal["map", "omezarr", "ometiff_image", "tiff_stack"]
+RawVolumeSourceT = Literal["map"]
 
+ChannelIdsMapping = dict[str, str]
+    
 
+class RawVolumeOptionsT(TypedDict):
+    """
+    Specifies the desired voxel size. Overwrites the automatically determined voxel size (e.g., the one based on the map header or its analogue for other formats).
+    Specifies the mapping of sequential channel IDs to user-defined ones.    
+    """
+    voxel_size: float | None
+    channel_ids_mapping: ChannelIdsMapping | None
+
+    
+class RawVolumeParams(BaseModel):
+    """
+    Create a volume from a parsed data resource based on the provided parameters
+    """
+
+    source: RawVolumeSourceT = Field(description="The type of the raw input file with volumetric data.")
+    options: RawVolumeOptionsT = Field(description="Specifies the voxel size and mapping of sequential channel IDs to user-defined channel IDs.")
+    
```

Comment:
I would prefer to keep the param hierarchy flat (without nesting) - it will be much easier to process params in frontend. Also it will allow adding `description` to each field separately.
I.e.:
```
    source: RawVolumeSourceT = Field(description="The type of the raw input file with volumetric data.")
    voxel_size: float | None = Field(...)
    channel_ids_mapping: ChannelIdsMapping | None = Field(...)
```